content/v14.x/en-US/doc/api/addons.md

```diff
@@ -154,25 +154,26 @@ they were created.
 
 The context-aware addon can be structured to avoid global static data by
 performing the following steps:
+
 * Define a class which will hold per-addon-instance data and which has a static
-member of the form
+  member of the form
   ```cpp
   static void DeleteInstance(void* data) {
     // Cast `data` to an instance of the class and delete it.
   }
   ```
 * Heap-allocate an instance of this class in the addon initializer. This can be
-accomplished using the `new` keyword.
+  accomplished using the `new` keyword.
 * Call `node::AddEnvironmentCleanupHook()`, passing it the above-created
-instance and a pointer to `DeleteInstance()`. This will ensure the instance is
-deleted when the environment is torn down.
+  instance and a pointer to `DeleteInstance()`. This will ensure the instance is
+  deleted when the environment is torn down.
 * Store the instance of the class in a `v8::External`, and
 * Pass the `v8::External` to all methods exposed to JavaScript by passing it
-to `v8::FunctionTemplate::New()` or `v8::Function::New()` which creates the
-native-backed JavaScript functions. The third parameter of
-`v8::FunctionTemplate::New()` or `v8::Function::New()`  accepts the
-`v8::External` and makes it available in the native callback using the
-`v8::FunctionCallbackInfo::Data()` method.
+  to `v8::FunctionTemplate::New()` or `v8::Function::New()` which creates the
+  native-backed JavaScript functions. The third parameter of
+  `v8::FunctionTemplate::New()` or `v8::Function::New()`  accepts the
```

Comment:
```suggestion
  `v8::FunctionTemplate::New()` or `v8::Function::New()` accepts the
```

Reply:
This is all generated from the upstream docs, so it would need to be addressed there. Anything changed here would be wiped out in the next sync